### PR TITLE
EAP6-345 deprecate web, securty & cmp subsystems

### PR DESCRIPTION
**Pull request by ctomc**
cmp, web, jacorb and messaging subsystems

### CODE REVIEW

**Comment by bstansberry** [comment]
What's the "sync resources with downstream" part of the 2nd commit? I don't see the effective difference.

That commit switches the handlers from the correct ones.

**Comment by ctomc** [comment]
@bstansberry in downstream (eap) we have newer model, with extra resource that is not present in upstream, main change is addition of UUIDKeyGeneratorResourceDefinition other stuff is just reorganization of code that was done in EAP but never in upstream.

File: legacy/cmp/src/main/java/org/jboss/as/cmp/subsystem/CMPSubsystemRootResourceDefinition.java [hunk 1]
```diff
@@ -24,39 +24,21 @@
...
         super(CmpSubsystemModel.SUBSYSTEM_PATH,
-                CmpExtension.getResolver(CmpExtension.SUBSYSTEM_NAME));
-    }
-
-    @Override
-    public void registerChildren(ManagementResourceRegistration subsystem) {
-
-        subsystem.registerSubModel(new ModelOnlyResourceDefinition(CmpSubsystemModel.UUID_KEY_GENERATOR_PATH,
-                CmpExtension.getResolver(CmpSubsystemModel.UUID_KEY_GENERATOR),
-                JNDI_NAME));
```
Comment (bstansberry) [inline legacy/cmp/src/main/java/org/jboss/as/cmp/subsystem/CMPSubsystemRootResourceDefinition.java]: Isn't this the UUID key generator stuff?

The second commit changes the type of the Extension from AbstractLegacyExtension, removing its behavior.

Comment (ctomc) [inline legacy/cmp/src/main/java/org/jboss/as/cmp/subsystem/CMPSubsystemRootResourceDefinition.java]: good catch, i missed that.

Comment (ctomc) [inline legacy/cmp/src/main/java/org/jboss/as/cmp/subsystem/CMPSubsystemRootResourceDefinition.java]: reversed the change.

**Pull request merged**